gnd/src/main/java/com/google/android/gnd/ui/home/featuresheet/ObservationListViewModel.java

```diff
@@ -76,22 +84,16 @@ public void loadObservationList(Feature feature, Form form) {
     return Single.just(ImmutableList.of());
   }
 
-  private void loadObservations(Project project, String layerId, String formId, String featureId) {
-    Optional<Form> form = project.getLayer(layerId).flatMap(pt -> pt.getForm(formId));
-    if (!form.isPresent()) {
-      // TODO: Show error.
-      return;
-    }
-    // TODO: Use project id instead of object.
+  private void loadObservations(Project project, Optional<String> formId, String featureId) {
```

Comment:
nit: Parallelize parameter positions? 

loadObservations:: project, formId, featureId
ObservationListRequest: project, featureId, formId

Swap feature and form in one or the other?

Reply:
Good catch. Done!